forge-gui/res/cardsfolder/upcoming/witchstalker_frenzy.txt

```diff
@@ -0,0 +1,7 @@
+Name:Witchstalker Frenzy
+ManaCost:3 R
+Types:Instant
+S:Mode$ ReduceCost | ValidCard$ Card.Self | Type$ Spell | Amount$ X | EffectZone$ All | Description$ This spell costs {1} less to cast for each creature that attacked this turn.
+SVar:X:Count$AttackersDeclared
```

Comment:
this only counts your own

Reply:
`SVar:X:Count$CreaturesAttackedThisTurn Creature` is not correct ?

Reply:
Same problem

Reply:
I guess Rowdy Research is scripted wrong as well. Any tips what the correct variable is here ?

Reply:
`PlayerCountPlayers$AttackersDeclared`